notebooks/Read_CoNLL-U_files.ipynb

```diff
@@ -0,0 +1,1116 @@
+{
```

Comment:
<p>This approach will work for now, but it's looking like we should have a proper "datasets" package to reduce the amount of duplicate code we're putting into all these notebooks. Marked that work down for later in https://github.com/CODAIT/text-extensions-for-pandas/issues/199</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>